lib/ansiblelint/utils.py

```diff
@@ -149,8 +149,11 @@ def find_children(playbook):
 
 def template(basedir, value, vars, fail_on_undefined=False, **kwargs):
     try:
-        value = ansible_template(os.path.abspath(basedir), value, vars, **kwargs)
-    except AnsibleError:
+        value = ansible_template(os.path.abspath(basedir), value, vars,
+                                 **dict(kwargs, fail_on_undefined=fail_on_undefined))
+        # Hack to skip the following exception when using to_json filter on a variable.
```

Comment:
I wouldn't mention to_json filter explicitly - it probably fails on all filters on the result of a variable!